process_pr.py

```diff
@@ -2096,15 +2108,20 @@ def process_pr(
                 if bot_status or (signatures["tests"] == "pending"):
                     new_bot_tests = True
                     trigger_test = True
-                    signatures["tests"] = "started"
+                    if not build_only:
+                        signatures["tests"] = "started"
```

Comment:
@iarspider , may be this is not needed? we override the test labels for `build_only` any way .... right?

Reply:
Yes.